pax_express_client/files/cli.py

```diff
@@ -110,3 +107,13 @@ def download(
         version=version,
         package=package,
     )
+
+
+@file_cli.command(name="delete", help="Delete a file")
+def delete(
+    repo: str = Option(..., "-r", "--repo"),
+    package: str = Option(..., "-p", "--package"),
+    version: str = Option(..., "-v", "--version"),
+    filename: str = Option(..., "-f", "--filename"),
```

Comment:
It would be nice if the filename would be optional - and if omitted it would fetch all file names and offer to delete all